### PR TITLE
fix: unbound variables are forbidden

### PR DESCRIPTION
**Pull request by blaggacao**
# Context

`disko` sets `set -u`. If the user forgets to set the flake attr (e.g. because the useage string doesn't suggest so), then an unbound error is raised.

# Solution

- default to nothing if unbound
- fix the usage string

### CODE REVIEW

**Comment by blaggacao** [comment]
@Lassulus this ended up becoming a scope-creep doc pr, please feel free to cherry pick at will. :handshake:

File: disko [hunk 1]
```diff
@@ -13,27 +13,35 @@ nix_args=()
...
+  where to mount the device tree (default: /mnt)
 * --dry-run
   just show the path to the script instead of running it
+* --no-deps
+  avoid adding another dependency closure to an in-memory installer
+    requires all necessary dependencies to be available in the environment
 * --debug
   run with set -x
+
+These are useful to skip copying executables uploading a script to an in-memory installer
```
Comment (Mic92) [inline disko:44]: Looks like a left over from --no-deps
```suggestion
```

[the remark this answers is not in the document]
Comment (blaggacao) [reply disko:44]: oh :smile: That was the prompt coming from a code comment that I had copied over. Sorry didn't notice I forgot to cleanup. :handshake:

**Review by Lassulus — APPROVED** [review]

**Comment by Lassulus** [comment]
@mergify queue

**Pull request merged**